manifests/Microsoft/VisualStudioCodeInsiders/1.46.0.yaml

```diff
@@ -0,0 +1,13 @@
+Id: Microsoft.VisualStudioCodeInsiders
+Version: 1.46.0
```

Comment:
This is technically not correct as the version is `x.y.z-insider`, but having the version number would require us to change it once a month. I was wondering in https://github.com/microsoft/winget-pkgs/issues/278 if we can use something like "Latest" or "Insider" here instead.